core/src/main/java/com/google/errorprone/bugpatterns/FallThrough.java

```diff
@@ -63,10 +65,16 @@ public Description matchSwitch(SwitchTree tree, VisitorState state) {
       // reported an error if that statement wasn't reachable, and the answer is
       // independent of any preceding statements.
       boolean completes = Reachability.canCompleteNormally(getLast(caseTree.stats));
+      int caseEndPositionIndex = caseEndPosition(state, caseTree);
+      int nextStartPositionIndex = next.getStartPosition();
+      // best effort work-around for https://github.com/google/error-prone/issues/613
+      if (caseEndPositionIndex == Position.NOPOS || nextStartPositionIndex == Position.NOPOS) {
```

Comment:
Did you see examples where `next.getStartPosition()` wasn't available, or just `caseEndPosition(state, caseTree)` ?